docusaurus/docs/ruby/rspec.mdx

```diff
@@ -65,23 +60,57 @@ Format stdout with the `documentation` formatter and file output with any RSpec
 
 ### Regular Mode
 
+You need to install the [`rspec_junit_formatter`](https://github.com/sj26/rspec_junit_formatter) gem.
+
 <Tabs>
   <TabItem value="junit-formatter" label="JUnit XML" default>
-    <XmlRegularModeFormatter />
+    ```bash
+    bundle exec rake "knapsack_pro:rspec[--format documentation --format RspecJunitFormatter --out tmp/rspec.xml]"
+    ```
   </TabItem>
   <TabItem value="json-formatter" label="JSON">
-    <JsonRegularModeFormatter />
+    ```bash
+    bundle exec rake "knapsack_pro:rspec[--format documentation --format json --out tmp/rspec.json]"
+    ```
   </TabItem>
 </Tabs>
 
 ### Queue Mode
 
+You need to install the [`rspec_junit_formatter`](https://github.com/sj26/rspec_junit_formatter) gem.
+
 <Tabs>
   <TabItem value="junit-formatter" label="JUnit XML" default>
-    <XmlQueueModeFormatter />
+    ```bash
+    bundle exec rake "knapsack_pro:queue:rspec[--format documentation --format RspecJunitFormatter --out tmp/rspec.xml]"
+    ```
+
+    You need to append the CI node index to the report name to avoid conflicts if your CI nodes write to the same disk:
+
+    ```bash
+    # Refer to your CI docs for `$MY_CI_NODE_INDEX`
+    export KNAPSACK_PRO_CI_NODE_INDEX=$MY_CI_NODE_INDEX
+
+    bundle exec rake "knapsack_pro:queue:rspec[--format documentation --format RspecJunitFormatter --out tmp/rspec_$KNAPSACK_PRO_CI_NODE_INDEX.xml]"
+    ```
+
+    This applies also if you are running parallel test processes on each CI node (see our page on to integrate Knapsack Pro with [`parallel_tests`](parallel_tests.md) for an example).
```

Comment:
Isn't this also true for Regular? Can we merge the Regular and Queue sections into one?

Maybe we could have separate sections for JUnit and JSON and use tabs for Queue and Regular (with Regular second).

I'd like to hide Regular vs Queue when possible.

Reply:
> Maybe we could have separate sections for JUnit and JSON and use tabs for Queue and Regular (with Regular second).
> 
> I'd like to hide Regular vs Queue when possible.

Good idea. Done: https://github.com/KnapsackPro/docs.knapsackpro.com/pull/152/commits/23f72c454c219b8625063908d6d3d193b55272d7

Reply:
> Isn't this also true for Regular? Can we merge the Regular and Queue sections into one?

We should not merge this. Additionally, recommending the use of Regular Mode with parallel_tests is not advisable.

Technically, you can use Regular Mode with the parallel_tests gem, but it's not practical to recommend this approach. For even distribution of tests with parallel_tests, a dynamic test split (Queue Mode) is necessary. Queue Mode is simply superior.